Intersect.Client/Interface/Game/Crafting/RecipeItem.cs

```diff
@@ -121,8 +121,7 @@ void pnl_HoverEnter(Base sender, EventArgs arguments)
             if (mIngredient != null && ItemBase.Get(mIngredient.ItemId) != null)
             {
                 DescWindow = new ItemDescriptionWindow(
-                    ItemBase.Get(mIngredient.ItemId), mIngredient.Quantity, mCraftingWindow.X, mCraftingWindow.Y,
-                    new int[(int) Stats.StatCount]
+                    ItemBase.Get(mIngredient.ItemId), mIngredient.Quantity, mCraftingWindow.X, mCraftingWindow.Y, null
```

Comment:
Change to `if (mIngredient != null && ItemBase.TryGet(mIngredient.ItemId, out var itemDescriptor))`

and pass `itemDescriptor` instead of doing a second `Get()` on this line.